lib/product/index.js

```diff
@@ -1,47 +1,53 @@
 'use strict';
 
-const _       = require('lodash'),
-      sqlite3 = require('sqlite3'),
-      db      = new sqlite3.Database('prod.db');
+var _       = require('lodash'),
+    sqlite3 = require('sqlite3'),
+    cache   = require('../../cache'),
+    db      = new sqlite3.Database('prod.db');
 
 module.exports = function() {
     function isNumeric(n) {
         return !isNaN(parseFloat(n)) && isFinite(n);
     }
 
     function getFlatProduct(req, res) {
-        const id     = req.params.pid,
-              locale = req.params.locale || 'en';
+        var id     = req.params.pid,
+            locale = req.params.locale || 'en';
 
-        let name, description, stock, price;
+        var name, description, stock, price;
+        
+        cache.get(id, function(error, result) {
```

Comment:
Shouldn't we handle the error and return the DB contents instead of the cached ones? That different in case the `result` is not set, right?